File: src/main/java/fr/epyi/metropiamod/gui/CustomInventoryContainer.java
package fr.epyi.metropiamod.gui;

import net.minecraft.entity.player.PlayerEntity;
import net.minecraft.inventory.container.Container;
import net.minecraft.inventory.container.Slot;

public class CustomInventoryContainer extends Container {

    public CustomInventoryContainer(int windowId, PlayerEntity player) {
        super(null, windowId);
    }

    @Override
    public boolean canInteractWith(PlayerEntity playerIn) {
        return false;
    }
}

File: src/main/java/fr/epyi/metropiamod/gui/CustomInventoryContainer.kt
package fr.epyi.metropiamod.gui

import net.minecraft.entity.player.PlayerEntity
import net.minecraft.inventory.container.Container

class CustomInventoryContainer(windowId: Int, player: PlayerEntity?) :
    Container(null, windowId) {
    override fun canInteractWith(playerIn: PlayerEntity): Boolean {
        return false
    }
}